### PR TITLE
add beta terms how to

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

A basic how-to for accepting beta terms.

Related to https://github.com/camunda/product-hub/issues/1838

## When should this change go live?

- [x] This change is not yet live and should not be merged until **mid November** (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
**Changes will be added to the current version of documentation once pre-approved.**
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
A few questions I am still looking to answer:

1. Where is the best placement for this document? Currently, it is just resting within Guides. Is this better suited within Console alongside managing users?
2. I still need to incorporate screenshots. Who is best to work with to obtain these, and ensure they are automated properly? CC @ultraschuppi
3. Is there a link to the Beta terms outside of the Console view?
4. To confirm, is this change live? Should this live within the current docs as well?
5. We note that "The system will confirm your acceptance and send a copy of the accepted beta terms to your email address." -- is this the address associated with the Camunda account holder? Are any other parties/users notified of this acceptance?

**Comment by UmmuFallon** [comment]
Hi Christina,

I will leave Eric and Collin to answer most of these questions as these are more directed to them.

For question 5, the beta terms will be sent to the email address that is associated with the Camunda account holder and currently the person that receives a cc is me (via Ironclad Clickwrap). There is a mixpanel notification, but Collin will need to confirm if there are additional users that will be notified.




Kind regards,



Ummu Fallon

Legal Operations Manager

Pronouns: She/her




[Camunda Logo]<https://camunda.com/>





Camunda Services GmbH

Zossener Str. 55–58

10961 Berlin



Amtsgericht Charlottenburg: HRB 113230 B

Geschäftsführer: Jakob Freund, Clemens Morgenroth

________________________________

**Review by HanselIdes — COMMENTED** [review]
Hi @christinaausley,

Let's pause this ticket for now.

After reviewing the feature you've documented, we've sent it back to design to complete the user flow and ensure that all desired features are present.

After the final iteration is completed, then I'll loop around with comments describing how the user experience has changed. I aim to get this done within Q3.

~Eric

**Comment by HanselIdes** [comment]
@christinaausley 

You can see the new user flow as part of my acceptance review in https://camunda.slack.com/canvas/C0563ESV2EL

The basic steps are:
1. Open Console > Organization Management > Settings
2. Opt in to the Beta terms (+ receive an email)
3. Open the documentation link to learn more about the feature
4. Toggle the feature

FYI here's a screenshot with the beta terms acceptance email image

To answer your questions:
> Where is the best placement for this document? Currently, it is just resting within Guides. Is this better suited within Console alongside managing users?

I think this belongs in Console documentation (cc @theburi ) under Organization Management, but it should also be linked from the [Early Access page](https://docs.camunda.io/docs/next/reference/early-access/). The link should explain how Enterprise customers can opt-in to certain early access features. **IMPORTANT: This is only relevant for Enterprise customers. Trial and professional users have early access features automatically enabled**

> I still need to incorporate screenshots. Who is best to work with to obtain these, and ensure they are automated properly? CC @ultraschuppi

@ColRad is the engineering DRI for this.

> Is there a link to the Beta terms outside of the Console view?

Not as far as I know. @UmmuFallon, does it make sense to add this to https://legal.camunda.com/?

> To confirm, is this change live? Should this live within the current docs as well?

This change happens after the 8.3 release. It should not be backdated

> Are there any additional individuals I should request review, or additional eyes we would like to see on this?

@camundom and I are good reviewers.

At this point, there will only minor changes to look and feel. You are unblocked to begin working on this. I would like the documentation available by November 3rd. Does that work for you?

**Comment by UmmuFallon** [comment]
> > Is there a link to the Beta terms outside of the Console view?
> 
> Not as far as I know. @UmmuFallon, does it make sense to add this to https://legal.camunda.com/?

Yes, this will be published in the legal center site once you are all ready to have this in production. Currently the beta terms in sandbox has it's own legal center and can be found [here](https://vault.pactsafe.io/s/22ae9332-9f93-4097-a688-3c702276c40e/beta-terms-test1.html) 

Either I publish on the Camunda Legal Center or I can create a new legal center specific for the beta terms, similar to the sandbox.

**Comment by HanselIdes** [comment]
@UmmuFallon creating a new legal center would probably not be a good use of your time. The docs only need a URL to point to, but the link can stay a placeholder until it's time to release the feature. If you know what the URL should be in advance, then please share it here. That can save in back-and-forth during the critical few hours of the release

**Comment by UmmuFallon** [comment]
> @UmmuFallon creating a new legal center would probably not be a good use of your time. The docs only need a URL to point to, but the link can stay a placeholder until it's time to release the feature. If you know what the URL should be in advance, then please share it here. That can save in back-and-forth during the critical few hours of the release

Sure, the URL will be https://legal.camunda.com/#beta-terms

**Comment by christinaausley** [comment]
@HanselIdes Thanks for all of the details here -- I'll get a draft together next week 👍

**Comment by akeller** [comment]
@christinaausley, just some quick feedback as this PR restarts:

* Move this content to the Console section
* Clarify this is for Enterprise only (we should have a badge/label for this but I'm not seeing it on our [meta page](https://docs.camunda.io/meta/#badges)).
  * However, ensure its clear that trial and starter plan (fka professional plan) users see these features automatically 
* Add the file to the sidebars
* Backport to 8.3, although "after the 8.3 release" has me confused. We only have the concept of 8.3 or our preview for 8.4 (`/next/`)

I updated the description to include the Nov 3 due date.

File: docs/guides/beta-terms.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+:::note
+Enabling beta is limited to [admin users](../components/console/manage-organization/manage-users.md) of Camunda products.
+:::
+
+## How to accept beta terms
+
+To accept beta terms for Camunda products, follow the steps below:
+
+1. Log in to Camunda Console and click the **Organization** tab to view the overview for **Organization Management**.
+   1. If the **Pre-Release Features** is marked as **Disabled**, you do not currently have admin access rights to the Camunda beta features. Visit our documentation on [managing users](../components/console/manage-organization/manage-users.md) to learn more.
```
Comment (ColRad) [inline docs/guides/beta-terms.md:27]: In the latest Iteration the Opt-In Button moved to the `Settings` tab in `Organization Management` 
The terms are now displayed inside a modal and the user can then toggle individual features. 
image
image
image

**Comment by christinaausley** [comment]
> Backport to 8.3, although "after the 8.3 release" has me confused. We only have the concept of 8.3 or our preview for 8.4 (/next/)

@HanselIdes can you confirm this please?

**Comment by christinaausley** [comment]
@ColRad @HanselIdes Can you please confirm or clarify the following?

- Is this still limited only to Admin users?
- For tier plans is this also automatically enabled?
- I noticed some of the UI has changed. Is there a way I can examine this live myself? For example, I want to know if there is still a way to see if you have access rights to the beta features. Do you still need to scroll to the bottom of the modal to accept the terms?

@ColRad -- once all of these are answered, could you please add the first screenshot of the latest iteration showing what is within the Settings tab?

**Comment by ColRad** [comment]
Hey @christinaausley This feature is currently enabled on DEV/INT ... as long as you have an Enterprise Org and the admin or owner role you can try it out yourself. Hit me up on Slack if you need an Org for testing. 
Also it looks like the wording changed from Beta -> Alpha yesterday, see https://camunda.slack.com/archives/C0589TC88UU/p1698240706114239?thread_ts=1697713380.992599&cid=C0589TC88UU

**Comment by HanselIdes** [comment]
> Backport to 8.3, although "after the 8.3 release" has me confused. We only have the concept of 8.3 or our preview for 8.4 (/next/)

I was mistaken. This should be backdated. Since it's in Console and all customers use the current version of Console, do you think it makes sense to put it in all versions?

>Is this still limited only to Admin users?

Only Admin and Owner users can view this feature

>For tier plans is this also automatically enabled?

This is for users on an Enterprise plan.

**Comment by christinaausley** [comment]
Hi all, this is ready for final review. Once approved, I will backport to `8.1`.

File: docs/components/console/manage-organization/alpha-terms.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+id: alpha-terms
+title: Alpha terms
```
Comment (akeller) [inline docs/components/console/manage-organization/alpha-terms.md:3]: Could we adjust the title to something more product-oriented and less legal-sounding? "Enable alpha features" or similar? 

"Alpha terms" sounds like something I would find in the footer of a website next to the policies.

@HanselIdes is there any relation to "alpha releases" (our monthly release assets)?

File: docs/components/console/manage-organization/alpha-terms.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+:::
+
+An alpha test is the second phase of software testing in which a sampling of the intended audience can test the product. When Camunda refers to **alpha**, we reference anything pre-release, such as any version of the software designated as alpha, limited release, developer preview, non-production, or evaluation.
+
+Alpha terms typically refer to the specific terms and conditions that govern the use and testing of this software during its alpha phase. These terms outline the rights, responsibilities, and limitations of both the software provider (Camunda) and the users (alpha testers or early adopters) during the testing and evaluation period.
+
+Alpha terms help protect Camunda´s interests (such as protecting our intellectual property, disclaiming warranties, or limiting our liability for any issues or damages that may arise during the alpha phase), manage user expectations, encourage active participation and feedback, and ensure legal compliance during the pre-release phase of software development.
+
+:::note
+Enabling alpha is limited to [admin users and owners](/manage-users.md) of Camunda products.
```
Comment (akeller) [inline docs/components/console/manage-organization/alpha-terms.md:19]: Should this be "alpha terms"?

File: docs/components/console/manage-organization/alpha-terms.md (hunk 2 above)
Comment (HanselIdes) [reply docs/components/console/manage-organization/alpha-terms.md:3]: @akeller this page refers to the program to enable alpha features in Console and Modeler. Alpha Terms is probably too legal-ese for it

File: docs/components/console/manage-organization/alpha-terms.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+id: alpha-terms
+title: Alpha terms
+description: "Learn how to launch the alpha terms agreement as an admin user or owner during the pre-release phase of software development. "
```
Comment (HanselIdes) [inline docs/components/console/manage-organization/alpha-terms.md:4]:
```suggestion
description: "Learn how admin users on Enterprise plans can gain early access to alpha features in Console and Modeler. "
```

File: docs/components/console/manage-organization/alpha-terms.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+3. Note the **Alpha Terms** modal. As the admin accepting the alpha terms, you must scroll and read through the terms prior to accepting.
+4. Once you have read the terms and scrolled through the modal, tick the box at the bottom reading **"I understand and agree to Alpha Terms"**.
+
+The system will confirm your acceptance and send a copy of the accepted alpha terms to your email address.
+
+## Utilize alpha features
+
+Once you accept the alpha terms, you can enable and disable any features you would like to use, and learn more about them.
+
+In the **Settings** tab, toggle the switch under **Status** to enable and disable the feature.
```
Comment (HanselIdes) [inline docs/components/console/manage-organization/alpha-terms.md:37]: Admins can also see when someone accepted the alpha terms and when features were enabled or disabled through the Activity log (see example [here](https://camunda.slack.com/archives/C0563ESV2EL/p1698143859194019))

**Review by HanselIdes — CHANGES_REQUESTED** [review]

File: docs/components/console/manage-organization/alpha-terms.md (hunk 2 above)
Comment (christinaausley) [reply docs/components/console/manage-organization/alpha-terms.md:3]: Adjusted in latest commit 👍

File: docs/components/console/manage-organization/alpha-terms.md (hunk 5 above)
Comment (christinaausley) [reply docs/components/console/manage-organization/alpha-terms.md:37]: Added in latest commit 👍

File: docs/components/console/manage-organization/alpha-terms.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+---
+id: alpha-terms
```
Comment (HanselIdes) [inline docs/components/console/manage-organization/alpha-terms.md:2]: I recommend also changing the filename and ID for consistency. The goal isn't to talk about terms, but how to get cool new features

File: docs/components/console/manage-organization/alpha-terms.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+The system will confirm your acceptance and send a copy of the accepted alpha terms to your email address.
+
+## Utilize alpha features
+
+Once you accept the alpha terms, you can enable and disable any features you would like to use, and learn more about them:
+
+- In the **Settings** tab, toggle the switch under **Status** to enable and disable the feature.
+- Click **View docs** under **Documentation** to learn more about the feature.
+- Admins are notified when someone accepts the alpha terms and when features are enabled or disabled under the **Activity** tab in **Organization Management**.
```
Comment (HanselIdes) [inline docs/components/console/manage-organization/alpha-terms.md:39]:
```suggestion
- Admins can know when someone accepts the alpha terms and when features are enabled or disabled under the **Activity** tab in **Organization Management**.
```
Notified implies the product actively pinging the admin.

File: optimize_sidebars.js [hunk 8]
```diff
@@ -68,6 +68,10 @@ module.exports = {
               "View organization activity",
               "components/console/manage-organization/view-organization-activity/"
             ),
+            docsLink(
+              "Enable alpha features",
+              "components/console/manage-organization/alpha-terms/"
```
Comment (HanselIdes) [inline optimize_sidebars.js:73]: see comment above about "alpha-terms"

**Review by HanselIdes — COMMENTED** [review]
I made a few comments to clean up the terminology

**Review by HanselIdes — CHANGES_REQUESTED** [review]
The decision to simplify to the alpha program also impacts the early access page. I've requested a few changes to make it consistent. 

I've also talked with the Play team, and the docs should not go live before at least 10 November, as we have some more work to do.

**Comment by christinaausley** [comment]
> The decision to simplify to the alpha program also impacts the early access page. I've requested a few changes to make it consistent.
> 
> I've also talked with the Play team, and the docs should not go live before at least 10 November, as we have some more work to do.

@HanselIdes I've linked between the two pages in my latest commit. As far as updating the `early-access` documentation, I think that is out of scope for this particular PR. Is someone from your team planning to update this documentation once the changes are complete?

I'm applying a hold label until we are ready to merge this.

**Comment by akeller** [comment]
Based on conversations on the epic, I'm not sure this is correct:

`This change is not yet live and should not be merged until Nov 3 (apply hold label or convert to draft PR)?`

For the up-to-date status, please see the epic comments - https://github.com/camunda/product-hub/issues/1838

**Comment by HanselIdes** [comment]
> @HanselIdes I've linked between the two pages in my latest commit. As far as updating the `early-access` documentation, I think that is out of scope for this particular PR. Is someone from your team planning to update this documentation once the changes are complete?

That's not an issue. I'll open a PR to change the early access docs

**Comment by ColRad** [comment]
@HanselIdes @akeller Can we merge this PR then?

**Comment by HanselIdes** [comment]
> @HanselIdes @akeller Can we merge this PR then?

@ColRad I added the content for the Play section as an add-on commit. That should simplify the merging process.

@akeller This can be merged and released with your next opportunity (NLT 15 Nov)

**Comment by HanselIdes** [comment]
> @HanselIdes Is there a reason why this isn't included in the 8.2 docs?

@christinaausley thanks for flagging. I didn't include it because I didn't want to confuse people by describing future features. However, telling people that it does support 8.2 features might be helpful. I wasn't confident in that choice, so please add it if you think it makes the docs better.

**Review by HanselIdes — APPROVED** [review]

**Comment by christinaausley** [comment]
Subsequent changes to come from @HanselIdes to clean up docs -- will merge in this follow-up PR and push out a release this morning 👍

**Pull request merged**